spec/cc/engine/analyzers/node_translator_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "spec_helper"
+require "cc/engine/analyzers/node_translator"
+
+RSpec.describe(CC::Engine::Analyzers::NodeTranslator) do
```

Comment:
Totally forgot about `CC::Parser::NodeTranslator`. My thinking was simple: the class translates a node. I don't necessarily think it implies `Node -> Node` but I see your point. Happy to change it.